packages/vite-node/src/cli.ts

```diff
@@ -74,11 +75,21 @@ async function run(files: string[], options: CliOptions = {}) {
     cli.version(version).outputHelp()
     process.exit(1)
   }
+  const root = options.root || process.cwd()
 
   const serverOptions = options.options
     ? parseServerOptions(options.options)
     : {}
 
+  serverOptions.deps ??= {}
```

Comment:
This will only work for an entry file. Here is how it's done in Vitest when using vite-node:

https://github.com/vitest-dev/vitest/blob/41e828f59c63d39e7b40ace0d226b37b0a1e3d52/packages/vitest/src/node/config.ts#L17

Reply:
So I should handle this in the vite node server? I'm not sure if my solution is correct

Reply:
Yes, you should handle it in vite node server